src/editor/components/modals/revision-diff.jsx

```diff
@@ -29,11 +50,43 @@ class RevisionDiffUnconnected extends React.Component {
     }
     return (
       <DiffContainer>
+        <RevertModal
+          visible={this.state.pendingRevert !== false}
+          onCloseOrCancel={() => {
+            this.setState({ pendingRevert: false });
+          }}
+          onRevert={() => {
+            this.props.revertToSelectedRevisionId();
+          }}
+          date={
+            this.props.selectedRevision
+              ? this.props.selectedRevision.created
+              : undefined
+          }
+        />
         <ReactDiffViewer
           oldValue={this.props.previousRevisionContent}
           newValue={this.props.currentRevisionContent}
           splitView={false}
         />
+        {this.props.canRevert && (
+          <Fab
+            onClick={() => {
+              this.setState({ pendingRevert: true });
+            }}
+            variant="extended"
+            style={{
+              right: 32,
+              bottom: 16,
+              position: "fixed",
+              color: "#fff",
+              background: THEME.clientModal.background,
+              borderRadius: 8
+            }}
+          >
+            Revert
```

Comment:
I would be in favor of using _restore_ as the word to use here, not revert. "Revert" implies undoing all the history that came after a revision, whereas "restore" implies what this functionality is doing – restoring the revision as the most recent one, preserving the rest of the revisions as history.

As such, I would suggest the text here "restore this version", a la Google Docs.

More broadly, I understand that would require going through these files and renaming all the `revert` parts to "restore`. In the long run I think it'll be clearer terminology.

Reply:
Made the files match the new terminology